core/constants.lua

```diff
@@ -137,6 +137,25 @@ const.EXPANSION_TYPE = {
   LE_EXPANSION_DRAGONFLIGHT = 9,
 }
 
+---@enum TooltipDataItemBinding
+const.TOOLTIPDATA_ITEM_BINDING = {
```

Comment:
This should read TOOLTIP_DATA_ITEM_BINDING

Reply:
Actually, looking at this again, do we have a particular reason we want to expose the fact this data originated from the tooltip? Why not ITEM_BINDING in this and all other enums?

Reply:
Seemed to me like pulling in the Blizzard Enums was the right choice for maintainability
https://github.com/Gethe/wow-ui-source/blob/live/Interface/AddOns/Blizzard_APIDocumentationGenerated/TooltipInfoSharedDocumentation.lua.